services/scan/src/store.ts

```diff
@@ -620,19 +675,28 @@ export class Store {
     );
   }
 
+  resetVotingState(): void {
+    if (this.hasElection()) {
+      this.client.run(
+        'update election set cvrs_backed_up_at = null, scanner_backed_up_at = null'
+      );
+      this.setPollsState('polls_closed_initial');
```

Comment:
This is unlikely to fail, but consider doing this as a transaction?

Reply:
For this case, can just do it in one statement. Maybe we will add more here later and it will need to be a transaction.

Reply:
Never mind, transactions are easy :)